bbmri/modules/directory-sync-compose.yml

```diff
@@ -6,7 +6,11 @@ services:
     environment:
       DS_DIRECTORY_URL: ${DS_DIRECTORY_URL}
       DS_DIRECTORY_USER_NAME: ${DS_DIRECTORY_USER_NAME}
-      DS_DIRECTORY_PASS_CODE: ${DS_DIRECTORY_PASS_CODE}
+      DS_DIRECTORY_USER_PASS: ${DS_DIRECTORY_USER_PASS}
       DS_TIMER_CRON: ${DS_TIMER_CRON}
       # It is recommended to check the enabling of this flag with your local data protection group
```

Comment:
Noone will find this here and it's too vague to be helpful. Rather, describe what's happening in the readme or where the service is enabled (where IS it enabled?)

Reply:
You say:

_Noone will find this here_

I am not sure I understand, could you give more details on the context of your statement?

The service is enabled in /etc/bridgehead/bbmri.conf.

Documented in: https://github.com/samply/bridgehead/?tab=readme-ov-file#directory-sync-tool